devices_policy.go

```diff
@@ -54,3 +122,117 @@ func (api *API) GetDeviceClientCertificatesZone(ctx context.Context, zoneID stri
 
 	return result, err
 }
+
+// CreateDeviceSettingsPolicy creates a settings policy against devices that match the policy
+//
+// API reference: TBA.
+func (api *API) CreateDeviceSettingsPolicy(ctx context.Context, accountID string, req DeviceSettingsPolicyRequest) (DeviceSettingsPolicyResponse, error) {
+	uri := fmt.Sprintf("/%s/%s/devices/policy", AccountRouteRoot, accountID)
+
+	result := DeviceSettingsPolicyResponse{}
+	res, err := api.makeRequestContext(ctx, http.MethodPost, uri, req)
+	if err != nil {
+		return result, err
+	}
+
+	if err := json.Unmarshal(res, &result); err != nil {
+		return result, fmt.Errorf("%s: %w", errUnmarshalError, err)
+	}
+
+	return result, err
+}
+
+// UpdateDefaultDeviceSettingsPolicy updates the default settings policy for an account
+//
+// API reference: TBA.
```

Comment:
similar to https://github.com/cloudflare/cloudflare-go/pull/1090/files#r979471255